content/blog/2024-05-21-Audit.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Security Audit for Eclipse Kuksa released"
+date = "2024-05-21"
+tags = ["KUKSA.val", "Audit", "Security"]
+categories = ["Implementation"]
+banner = "img/banners/2024-05-21-auditBanner.png"
++++
+
+Applications that utilize artifacts from the Kuksa project need to rely on the secure execution of these components. We are, therefore, happy to announce the publication of an external audit report carried out by [Quarkslab](https://www.quarkslab.com). The audit was facilitated through [OSTIF](https://ostif.org) and the [Eclipse Foundation](https://www.eclipse.org) and made possible by the founding the Eclipse Foundation received from the [Alpha-Omega project](https://openssf.org/community/alpha-omega/).
+
+- The audit was scoped on the KUKSA databroker and the respective Python client SDK.
+- There were 19 findings with low or higher severity, of which two had a high severity.
```

Comment:
There were 19 findings, of which 17 had low and two had high severity.

Reply:
There were 2 High, 1 Medium, 10 Low and 6 Info findings. For the summary I thought this already too much detail. I changed it to only mention the total number of the number high findings.